Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.yml

```diff
@@ -1627,7 +1627,20 @@ script:
       will not update the current incident, it's here for debugging purposes.
     execution: false
     name: get-remote-data
-  dockerimage: demisto/python3:3.8.6.12176
+  - arguments:
+    - default: false
+      description: Date string representing the local time.The incident is only returned
+        if it was modified after the last update time.
+      isArray: false
+      name: lastUpdate
+      required: false
+      secret: false
+    deprecated: false
+    description: Get the list of incidents that were modified since the last update.
+      Please note that this method is here for debugging purposes.
```

Comment:
```suggestion
    description: Get the list of incidents that were modified since the last update.
      Please note that this method is here for debugging purposes. get-modified-remote-data is used as part of a Mirroring feature, which is available since version 6.1.
```